crates/sui/src/upgrade_compatibility.rs

```diff
@@ -813,6 +821,47 @@ fn module_compatibility_error_diag(
     Ok(diags)
 }
 
+const PACKAGE_TABLE: &str = "[package]";
```

Comment:
nit: you can move this inside the function body if this is the only place this constant is used (or to the top of the file if you plan on using it in more places).